README.md

```diff
@@ -80,26 +80,44 @@ The quickest and easiest way to install the software is to download the most rec
+## Verifying Your Software (optional, but recommended!)
+You can verify the data integrity and authenticity of the software with as little as three commands. This process assumes that you know [how to navigate on a terminal](https://terminalcheatsheet.com/guides/navigate-terminal) and have navigated to the folder where you have these three relevant files present: (This will most likely be your Downloads folder.)
 
-* seedsigner_pubkey.gpg (from the main folder of this repo)
-* seedsigner_0_4_6.img.zip (from the software release)
-* seedsigner_0_4_6.img.zip.sha256 (from the software release)
-* seedsigner_0_4_6.img.zip.sha256.sig (from the software release)
 
-**Note:** The specific version number of the files in your folder might not match the above exactly, but their overall format and amount should be the same.
+* seedsigner_0_5_x.img.zip (from the software release)
+* seedsigner_0_5_x.img.zip.sha256 (from the software release)
+* seedsigner_0_5_x.img.zip.sha256.sig (from the software release)
 
-This process also assumes you are running the commands from a system where both [GPG](https://gnupg.org/download/index.html) and [shasum](https://command-not-found.com/shasum) are installed and working.
+**Note:** The specific names of the 3 files might not match the above examples exactly, but their naming format will be the same.
 
-First make sure that the public key is present in your keychain:
-```
-gpg --import seedsigner_pubkey.gpg 
-```
-This command will import the public key, or return:
+This process also assumes you are running the commands from a system where both [GPG](https://gnupg.org/download/index.html) and [shasum](https://command-not-found.com/shasum) are already installed and working.
+
+First you will add/update the *public key* of the SeedSigner project into your keychain. The command below will fetch our public key from a popular online keyserver called *Keybase.io* , and once the Key is imported successfully, we will visually compare its properties to that website.
 ```
-key <...> not changed
+gpg --fetch-keys https://keybase.io/SeedSigner/pgp_keys.asc
 ```
+Once the command completes successfully, it will display a numeric ID, as circled red in the example below. We will use it in the next step.
+![SS - Keybase PubKey import with Fingerprint shown (New import or update of the key)](https://user-images.githubusercontent.com/91296549/174248861-7961c038-1fbf-47a1-a110-146cb218b1c8.jpg)
+
+You should now compare the numeric ID  which your computer just provided you, to what is displayed on this website 
+[Keybase.io/seedsigner](https://www.keybase.io/SeedSigner)
+
+These numeric ID's are known as the Key's *fingerprint*, so please make sure that the fingerprints **do** match. (The white spaces doesnt matter, its there to help readability.) 
```

Comment:
We are definitely not using apostrophes for plural!!! "These numeric IDs are..."

Also: consider the parenthetical as part of the sentence: "...do match (blah... readability)." Period goes at the very end, outside the parentheses and the inner text should omit its period.

Reply:
Many thanks Keith, I will fix those and look for others.  I have made a few more edits too and will push those upstream. Do you have a recommendation for a MD editor? Github.com is getting very tiring for long edits like this

Reply:
I mainly edit markdown in VSCode. Github actually has a web-based VSCode feature: Just type period on the repo's root github page. Pretty crazy. A little hard to use at first if you're not used to VSCode, though. You'll want to watch an intro tutorial for how the UI integrates with git to submit new changes. I don't recall the details, but there's a way to preview the rendered markdown, too.

Reply:
@kdmukai Keith !! press period....holy macaroni, a whole new environment just appeared !!! 
toot toot!!  that's amazing, thank v v much! 
so much easier now!